dsgrid/config/date_time_dimension_config.py

```diff
@@ -175,7 +179,26 @@ def get_time_ranges(self):
 
         return ranges
 
-    def get_load_data_time_columns(self):
+    def get_start_times(self) -> list[pd.Timestamp]:
```

Comment:
Why pd.Timestamp instead of Python datetime?

Reply:
Doesn't have to, I think it was because build_time_ranges was using pd.Timestamp and I was just following that convention.

Reply:
On second thought, I think it is the easiest to implement to instantiate from str-like input.

Reply:
Changed to use datetime